python/ray/_private/runtime_env/conda.py

```diff
@@ -96,9 +97,9 @@ def setup_conda_or_pip(runtime_env: dict,
         # lock for all conda installs.
         # See https://github.com/ray-project/ray/issues/17086
         file_lock_name = "ray-conda-install.lock"
-        with FileLock(os.path.join(context.session_dir, file_lock_name)):
-            conda_dir = os.path.join(context.session_dir, "runtime_resources",
-                                     "conda")
+        with FileLock(os.path.join(context.resources_dir, file_lock_name)):
+            conda_dir = os.path.join(context.resources_dir,
+                                     "runtime_resources", "conda")
```

Comment:
The directory layout isn't changed in this PR, right?  In particular `resources_dir` refers to`/tmp/ray/session_latest`, and `runtime_env_dir` refers to `/tmp/ray/session_latest/runtime_resources`?  But I also see things like `resources_dir=self._runtime_env_dir`.  Do we just need to delete the argument `"runtime_resources"` from this line?